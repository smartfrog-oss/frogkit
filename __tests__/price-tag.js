import { mount } from 'vue-test-utils'
import PriceTag from '@/components/price-tag'

describe('Price Tag component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(PriceTag, {propsData: {value: '9.59', code: 'EUR' }})
    expect(wrapper.html()).toMatchSnapshot()
  })

})
