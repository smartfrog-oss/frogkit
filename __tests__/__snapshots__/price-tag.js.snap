// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price Tag component should render component and match snapshot 1`] = `
<div class="fk-price-tag">
  <p class="fk-price-tag__price">
    <b>9,<sup class="fk-price-tag__decimal">59</sup></b>â‚¬
  </p>
</div>
`;
